contracts/proto/faas/v1/faas.proto

```diff
@@ -89,6 +89,23 @@ message ScheduleCron {
   string cron = 1;
 }
 
+// Notification Workers
+enum EventType {
```

Comment:
Better to be more specific with the naming of these enum e.g. `BucketNotificationType`